rust/ql/test/library-tests/variables/variables.expected

```diff
@@ -411,49 +411,49 @@ variableInitializer
 | variables.rs:37:9:37:10 | x3 | variables.rs:38:9:38:14 | ... + ... |
 | variables.rs:43:9:43:10 | x4 | variables.rs:43:14:43:16 | "a" |
 | variables.rs:46:13:46:14 | x4 | variables.rs:46:18:46:20 | "b" |
-| variables.rs:75:9:75:10 | p1 | variables.rs:75:14:75:37 | RecordExpr |
-| variables.rs:85:9:85:10 | s1 | variables.rs:85:14:85:41 | CallExpr |
-| variables.rs:102:9:102:10 | s1 | variables.rs:102:14:102:41 | CallExpr |
-| variables.rs:111:9:111:10 | x6 | variables.rs:111:14:111:20 | CallExpr |
+| variables.rs:75:9:75:10 | p1 | variables.rs:75:14:75:37 | Point {...} |
+| variables.rs:85:9:85:10 | s1 | variables.rs:85:14:85:41 | Some(...) |
+| variables.rs:102:9:102:10 | s1 | variables.rs:102:14:102:41 | Some(...) |
+| variables.rs:111:9:111:10 | x6 | variables.rs:111:14:111:20 | Some(...) |
 | variables.rs:112:9:112:10 | y1 | variables.rs:112:14:112:15 | 10 |
 | variables.rs:128:9:128:15 | numbers | variables.rs:128:19:128:35 | TupleExpr |
-| variables.rs:155:9:155:10 | p2 | variables.rs:155:14:155:37 | RecordExpr |
-| variables.rs:169:9:169:11 | msg | variables.rs:169:15:169:38 | RecordExpr |
-| variables.rs:189:9:189:14 | either | variables.rs:189:18:189:33 | CallExpr |
-| variables.rs:203:9:203:10 | tv | variables.rs:203:14:203:36 | CallExpr |
-| variables.rs:219:9:219:14 | either | variables.rs:219:18:219:33 | CallExpr |
-| variables.rs:229:9:229:14 | either | variables.rs:229:18:229:33 | CallExpr |
-| variables.rs:253:9:253:10 | fv | variables.rs:253:14:253:35 | CallExpr |
-| variables.rs:315:9:315:23 | example_closure | variables.rs:316:9:317:9 | ClosureExpr |
-| variables.rs:318:9:318:10 | n1 | variables.rs:319:9:319:26 | CallExpr |
-| variables.rs:323:9:323:26 | immutable_variable | variables.rs:324:9:325:9 | ClosureExpr |
-| variables.rs:326:9:326:10 | n2 | variables.rs:327:9:327:29 | CallExpr |
-| variables.rs:332:9:332:9 | v | variables.rs:332:13:332:41 | RefExpr |
+| variables.rs:155:9:155:10 | p2 | variables.rs:155:14:155:37 | Point {...} |
+| variables.rs:169:9:169:11 | msg | variables.rs:169:15:169:38 | Message::Hello {...} |
+| variables.rs:189:9:189:14 | either | variables.rs:189:18:189:33 | Either::Left(...) |
```

Comment:
I'm probably not the right person to review the extractor changes right now but I will say this: these look good, _much_ better than before.